### ISSUE TITLE
Ag grid row id grid assigned (auto generated by grid ) id logic.

### ISSUE THREAD

**Issue by srinath-varadan** [ceolter/ag-grid#7664]
Is there any internal methods to get the auto assigned row id (incremental order ) in getrow id call back params or any other workaround to access this id

App didn't have the get row id call back explicitly and the state of the grid is persisted in db. If get row id callback is introduced now with the leaf and row group logics mentioned in examples
 The expansion doesn't happens since the newly created row id doesn't match with the auto generated id which was persisted in db. Hence need a way to access the auto generated id in getrow id call back. 

Basically need to assign the same auto generated id in get row id call back function

**Comment by srinath-varadan** [comment]
Any update on this ? . Need the auto increment logic which is currently used if get row id is not defined .

**Comment by AG-Zoheil** [comment]
Thank you for submitting this on Github.

As this has been open without any further interaction for a while, I will be closing this issue.

If the issue has already been resolved by other means, you can ignore this. If however, you are still in need of support you can choose to raise the issue again for the AG Grid community to support.

Kind regards,
Zoheil
